src/components/ApplicationNameBar.tsx

```diff
@@ -9,14 +9,12 @@ const ApplicationNameBar = ({ text, href }: ApplicationNameBarProps) => {
   return (
     <div id="app-bar" className="bg-blue-dark">
       <section className="container mx-auto p-4">
-        <h2>
-          <Link
-            href={href}
-            className="font-body text-lg font-bold text-white underline md:text-[28px]"
-          >
-            {text}
-          </Link>
-        </h2>
+        <Link
```

Comment:
This one was previously requested to be an h2.

#4869

Reply:
Fixed